kahuna/public/js/components/gr-collections-panel/gr-collections-panel-node.html

```diff
@@ -37,7 +37,7 @@
                 title="Add a new sub-collection"
                 ng:if="ctrl.editing"
                 ng:click="$parent.active = !$parent.active">
-            <gr-icon>add_box</gr-icon>
+            <gr-icon>create_new_folder</gr-icon>
```

Comment:
If we're going for collection = folder, maybe we should replace the icon on the collection tab?